### DISCUSSION TITLE
Ресурсы по ТФЯ

### DISCUSSION THREAD

**Discussion by TonitaN** [TonitaN/TFL2021#33] [Ideas]
Последний месяц наверное можно объявить "месяцем ресурсов по ТФЯ": прирост ресурсов по ТФЯ для ИУ9 на github увеличился вдвое. Мне не очень понятно, согласуются ли все эти репозитории между собой и с трудами @desitas1701. Хотелось бы понять, насколько это централизованные процессы, и что имеет смысл отслеживать, а что нет.

**Comment by TonitaN** [comment]
@BaldiSlayer @HumsterProgrammer @ark2016 и собственно @desitas1701 (на всякий случай тегаю ещё @Tixy05 , у него вроде тоже какие-то мысли были, но не уверена, что пока только не в проекте)

**Comment by BaldiSlayer** [comment]
Хм, а у меня разве есть какой-то репозиторий с ресурсами по тфя?...

**Reply by TonitaN** [reply]
https://github.com/iu9-rofl
Надеюсь, это не под NDA)

**Reply by BaldiSlayer** [reply]
а, Вы про это

на самом деле это просто копия нашего репозитория с первой лабой, потому что я уже отчаялся при попытке починки той джобы и думал, что придется выносить в организацию


репозиторий @HumsterProgrammer и @ark2016 видел мельком)

про труды @desitas1701 не знаю, что именно имеется в виду
если говорится про записи лекций и семинаров, то да, знаю, потому что они лежат на моем Яндекс диске))

<img width="262" alt="Снимок экрана 2024-12-01 в 03 20 56" src="https://github.com/user-attachments/assets/a81ad3b8-23c0-4690-8815-ebe7b0ccdda3">

**Reply by BaldiSlayer** [reply]
> Надеюсь, это не под NDA)

аххахха, нет

все, что лежит у меня на гитхабе не под NDA))
даже `t-bmstu` (фан факт, у него есть около 3-4 копий на гитфлике в разных репозиториях, причем к части из них я не имею доступа)

**Reply by TonitaN** [reply]
Егор тоже пишет какие-то выдержки для БЗ (но они, как я поняла, под NDA).
Не хотелось бы, чтобы народ делал одну и ту же работу по разбору всяких невнятных ыавппапап с лекций несколько раз (да ещё и не автоматически).

**Reply by TonitaN** [reply]
>
> даже `t-bmstu` (фан факт, у него есть около 3-4 копий на гитфлике в разных репозиториях, причем к части из них я не имею доступа)

Т.е. он жив?

**Reply by BaldiSlayer** [reply]
> > даже `t-bmstu` (фан факт, у него есть около 3-4 копий на гитфлике в разных репозиториях, причем к части из них я не имею доступа)
> 
> Т.е. он жив?

на самом деле у него очень трагичная судьба. Он был жив какое-то время (я его поднял на выделенной для него Бауманской виртуалке), но им никто не пользовался и я туда не контрибьютил какое-то время, так как был занят (хотя на самом деле у меня есть много идей что туда сделать, но руки не доходят, а еще там нужно много что пофиксить). 
Потом им занимались первокурсники (уже второкурсники) в рамках летней практики. Они что-то сделали, но я не знаю что, потому что у них приватный репозиторий на гитфлике. Жаль. Они пропустили возможность получить от меня ревью....

Сейчас им вроде никто не занимается, да и нигде он не захосчен =(

**Comment by HumsterProgrammer** [comment]
Я не торопился анонсировать то, что мы делаем(я, @ark2016, и еще несколько человек) пока не определимся с форматом и не отработаем вообще процесс заполнения материалов.

Изначально была идея прорешать практические задания. Потом @ark2016 предложил также и теорию расписывать. На данный момент еще в сырой стадии, но вроде почти устаканился концепт.

Насчет централизованности - если друге похожие инициативы есть, то я про них не знал.

**Reply by TonitaN** [reply]
> 
> Изначально была идея прорешать практические задания. Потом @ark2016 предложил также и теорию расписывать. На данный момент еще в сырой стадии, но вроде почти устаканился концепт.
> 

Есть же ещё решения РК прошлых лет, да и в общем другие люди, затехавшие свои РК (опять привет @BaldiSlayer ). Неудобно, если всё это будет разрозненно. И лучше одному человеку не грузиться вообще всем сразу, и практикой, и теорией - а распределить, если уж спонтанно возникают такие инициативы.

P.S. - Очень вас не хватало вчера на лабах. Жду в следующий раз.

**Reply by ark2016** [reply]
я свои наработки добавил в репозиторий к @HumsterProgrammer, остальные идеи пока что не реализованы
